packages/map/components/Map.tsx

```diff
@@ -307,9 +308,25 @@ const Map = ({ serverId }: MapProps) => {
                         </LayerGroup>
                     </LayersControl.Overlay>
 
+                    <LayersControl.Overlay 
+                        checked={localStorage.getItem('layer-mainline-signals') === null || localStorage.getItem('layer-mainline-signals') === 'true'}
+                        name="Mainline signals">
+                        <LayerGroup>
+                            <MainlineSignals />
+                        </LayerGroup>
+                    </LayersControl.Overlay>
+
+                    <LayersControl.Overlay 
+                        checked={localStorage.getItem('layer-other-signals') === null || localStorage.getItem('layer-other-signals') === 'true'}
+                        name="Other signals">
+                        <LayerGroup>
+                            <OtherSignals />
+                        </LayerGroup>
+                    </LayersControl.Overlay>
+
```

Comment:
I'm not sure if it's a good idea to enable it by default.
I'm afraid that rendering too many markers on the map would cause some performance issues

What do you think ?

Reply:
Oops, my bad. Will boot up my laptop to fix